src/energyplus/ForwardTranslator/ForwardTranslatePlantEquipmentOperationSchemes.cpp

```diff
@@ -204,8 +206,14 @@ namespace energyplus {
         // Probably check if all modules have a ChillerHeaterPerformanceComponent
         // that has a flow rate hardsized, and multiple by the number of performance comp
         // Better to add a method to centralheatpumpsystem, for eg "totalDesignFlowRate"
-        // Problem is that you need to know which loop is requesting this since there are three
-        // "Design XXX Water Flow Rate" (XXX= Chilled Water, Condenser Water, or Hot Water)
+        // Problem is that you need to know which loop is requesting this since there are two supply loops with
+        // "Design XXX Water Flow Rate" (XXX= Chilled Water, or Hot Water)
+        break;
+      }
+
+      case openstudio::IddObjectType::OS_Chiller_Electric_ASHRAE205: {
+        auto chiller = component.cast<ChillerElectricASHRAE205>();
+        result = chiller.chilledWaterMaximumRequestedFlowRate();
```

Comment:
No shenanigans needed, it'll always be on the supply side of only one Plant Loop.